roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -80,15 +87,14 @@ kubernetesMasterConfig:
   - v1
   apiServerArguments: {{ api_server_args if api_server_args is defined else 'null' }}
   controllerArguments: {{ controller_args if controller_args is defined else 'null' }}
-{# TODO: support overriding masterCount #}
-  masterCount: 1
-  masterIP: ""
-  podEvictionTimeout: ""
+  masterCount: {{ openshift.master.master_count }}
+  masterIP: {{ openshift.common.ip }}
+  podEvictionTimeout: 5m
```

Comment:
Do we want to explicitly set podEvictionTimeout rather than using an empty string and allowing it to use the system default? (at least until we provide a way to override)